extension/src/background/index.ts

```diff
@@ -59,7 +60,9 @@ export const initInstalledListener = () => {
   browser?.runtime?.onInstalled.addListener(normalizeMigratedData);
   browser?.runtime?.onInstalled.addListener(migrateFriendBotUrlNetworkDetails);
   browser?.runtime?.onInstalled.addListener(migrateSorobanRpcUrlNetworkDetails);
-  browser?.runtime?.onInstalled.addListener(migrateTokenIdList); 
+  browser?.runtime?.onInstalled.addListener(migrateTokenIdList);
+  // migrate storage version last so that other migrations can compare versions
+  browser?.runtime?.onInstalled.addListener(migrateDataStorageVersion);
```

Comment:
Do these run sequentially? Should this be at the top? I'm not 100% sure but we should double check to be safe